charts/seed-bootstrap/charts/vpa/values.yaml

```
@@ -16,7 +16,8 @@ admissionController:
   podAnnotations: {}
   podLabels: {}
   enableServiceAccount: true
-  registerByURL: false
+  # caCert: abcd
```

Sorry, I just saw that it's no optional value. Let's revert it to:

```suggestion
  caCert: abcd
```